internal/server/statvar/golden/search_statvar/female.json

```diff
@@ -8579,174 +8749,6 @@
-      "name": "Student With Gender = Female, School Grade Level = School Grade 4"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolGradeLevel-SchoolGrade8",
-      "name": "Student With Gender = Female, School Grade Level = School Grade 8"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject",
-      "name": "Student With Gender = Female, School Subject"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Civics",
-      "name": "Student With Gender = Female, School Subject = Civics"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Economics",
-      "name": "Student With Gender = Female, School Subject = Economics"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Geography",
-      "name": "Student With Gender = Female, School Subject = Geography"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Mathematics",
-      "name": "Student With Gender = Female, School Subject = Mathematics"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Music",
-      "name": "Student With Gender = Female, School Subject = Music"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Reading",
-      "name": "Student With Gender = Female, School Subject = Reading"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Science",
-      "name": "Student With Gender = Female, School Subject = Science"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-TechnologyAndEngineeringLiteracy",
-      "name": "Student With Gender = Female, School Subject = Technology And Engineering Literacy"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-USHistory",
-      "name": "Student With Gender = Female, School Subject = USHistory"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-VisualArts",
-      "name": "Student With Gender = Female, School Subject = Visual Arts"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolSubject-Writing",
-      "name": "Student With Gender = Female, School Subject = Writing"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolType",
-      "name": "Student With Gender = Female, School Type"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolType-PrivateSchool",
-      "name": "Student With Gender = Female, School Type = Private School"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SchoolType-PublicSchool",
-      "name": "Student With Gender = Female, School Type = Public School"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SubjectAssessmentScale",
-      "name": "Student With Gender = Female, Subject Assessment Scale"
-    },
-    {
-      "dcid": "dc/g/Student_Gender-Female_SubjectAssessmentScale-AlgebraScale",
-      "name": "Student With Gender = Female, Subject Assessment Scale = Algebra Scale"
```

Comment:
I assume deletions are due to a limit on selected Svs for this file as the related import was not changed? 
The SVs and SVGs are present in both autopush and prod.